src/windows_aligned_file_reader.cpp

```diff
@@ -43,9 +45,17 @@ void WindowsAlignedFileReader::register_thread()
         FILE_ATTRIBUTE_READONLY | FILE_FLAG_NO_BUFFERING | FILE_FLAG_OVERLAPPED | FILE_FLAG_RANDOM_ACCESS, NULL);
     if (ctx.fhandle == INVALID_HANDLE_VALUE)
     {
-        std::stringstream ss;
-        ss << m_filename.c_str();
-        diskann::cout << "Error opening " << ss.str() << " -- error=" << GetLastError() << std::endl;
+        const size_t c_max_filepath_len = 256;
+        size_t actual_len = 0;
+        char filePath[c_max_filepath_len];
```

Comment:
use vector<char> to avoid allocating big array in stack?

Reply:
I think 256 bytes is not so big?